src/power_grid_model/validation/rules.py

```diff
@@ -697,6 +697,33 @@ def none_missing(data: SingleDataset, component: str, fields: Union[str, List[st
     return errors
 
 
+def exist_in_pairs(data: SingleDataset, component: str, field_1: str, field_2: str) -> bool:
+    """
+    Check if anyone of the specific pair of fields of a particular type of component exist.
```

Comment:
here and below: `anyone` is reserved for people. `any one` or just `any` would be correct since it concerns inanimate objects

in addition, the body doesn't do what the name suggests: `exist in pairs` to me suggests the `AND` operation, while the body and description suggest `OR`